### PR TITLE
🎥 adds video thumbnail generation #2169

### PR DESCRIPTION
**Pull request by rezk2ll**
## Description
- adds video thumbnail generation service in the node backend.
- adds the docker installation instruction for FFmpeg.
- shows the video thumbnail in the frontend.
## Related Issue
https://github.com/linagora/Twake/issues/2169

## Motivation and Context
- missing video preview

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added the Signed-off-by statement at the end of my commit message.

### CODE REVIEW

**Comment by rezk2ll** [comment]
> Nice work! Could we have a demo tomorrow in daily ? I'm interested to see what it looks like. Also maybe could we plan some changes in frontend for this new thumbnail, let's discuss it in daily.

small demo:

https://user-images.githubusercontent.com/6764881/169335511-83accbb8-d9d1-43b4-9803-77b069f01960.mp4

File: twake/backend/node/src/services/previews/services/processing/video.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  inputPath: string,
+  outputFolder: string,
+  outputFile: string,
+): Promise<void> => {
+  return new Promise((resolve, reject) => {
+    ffmpeg(inputPath)
+      .screenshot({
+        count: 1,
+        filename: outputFile,
+        folder: outputFolder,
```
[the remark this answers is not in the document]
Comment (rezk2ll) [reply twake/backend/node/src/services/previews/services/processing/video.ts:54]: by default, if we don't specify the `width` and `height` they default to `320x240`. so maybe we set a minimum values instead for a large enough preview.

[the remark this answers is not in the document]
Comment (RomaricMourgues) [reply twake/backend/node/src/services/previews/services/processing/video.ts:54]: 320x240 seems a bit small indeed, let's see how it looks when we implement the frontend in the work of #2138 then

**Comment by RomaricMourgues** [comment]
@rezk2ll I wait for the tests, tell me when it'll be ready to review+merge

**Pull request merged**